README.md

```diff
@@ -117,3 +118,132 @@ When you resize the VirtualBox window (e.g. to mazimise it) the display resoluti
+
+You should add the name of and full path to your key to your **_aws-credentials_** file. Note that the name should not include any extension (e.g. `.pem`), but the full path should.
+
+### Create a security group to limit IP access to your VM
+
+In the EC2 Dashboard scroll down the left-hand menu until you find **Network & Security** and click **Security Groups**, and then click **Create security group** on the top right. 
+
+You should give it a name, e.g. **MyIP** as is used in the Vagrantfile, and a description (e.g. "limit access to my IP"). Scroll down to the **Inbound rule** section and click **Add rule**.
+
+Here, use the drop-down menus to change the _Type_ to **All traffic** and the _Source_ to **My IP** (your current IP address will be automatically added). Scroll down to the bottom of the page and click **Create security group**.
+
+If you used a name other than "MyIP" for the name of the group you will need to update the `aws.security_groups` setting in the AWS Vagrantfile.
+
+### Create a user
+
+You will need to create a user with the correct permissions to access your EC2 VM, which again is done via the console. This is not done within the EC2 Dashboard, but is instead done within the **IAM Dashboard** (Identity and Access Management). To get to this from the EC2 Dashboard first click the AWS logo on the top left of the page to bring you back to the console front page, and then click the **All services** drop-down menu, and then click **IAM** under the "Security, Identity, & Compliance" section.
+
+Under **Access Management** click **Users** and then click **Add user**. You should give them a name, e.g. _vagrant_ or _metomi-vms_ etc.. Tick the box for **Programmatic access** and then click the **Next: Permissions** button.
+
+Here you should click the tab labelled **Attach existing policies directly** and search for **AmazonEC2FullAccess** and then tick the check-box next to this option. Now click the **Next: Tags** button. You can then click the **Next: Review** button. 
+
+Now click **Create user**. This will bring you to a page listing the username, the _Access key ID_ and the _Secret access key_. **THE SECRET ACCESS KEY INFORMATION WILL BE DISPLAYED ONLY ONCE**. 
+
+You should copy this information into your **_aws-credentials_** file and download and save the `.csv` file containing this information. Again, do not upload this information (either the aws-credentials file or the csv file) to a public repository.
+
+### Provision your AWS VM
+
+Once you have all the information for your aws-credentials file, you should first _source_ this file
+```
+source aws-credentials
+```
+Once you have created (& potentially added) the security group to your AWS Vagrantfile, you should edit the top-level Vagrantfile to point to `Vagrantfile.aws_ubuntu-1804`. Then provision the VM by
```

Comment:
Presumably this would work as well?
```
export VAGRANT_VAGRANTFILE=Vagrantfile.aws_ubuntu-1804
```
If you added this to `aws-credentials` this would avoid the need to edit `Vagrantfile`

Reply:
Thanks for the suggestion - yes, that works as well and makes things cleaner as it doesn't involve editing the Vagrantfile. I've incorporated it in my last change.

Reply:
I've tweaked the text which hopefully makes this clearer.